java/src/test/java/vowpalWabbit/learner/VWLearnersTest.java

```diff
@@ -67,6 +68,15 @@ public void testBadModel() {
     }
 
     @Test
+    public void testSaveModel() {
+        VWLearner vw = VWLearners.create("--quiet");
+        File file = new File("/tmp/saved_test_model");
```

Comment:
Just noticed this will leave a file on the user's file system after the test is run.  Tests should leave the machine in the same state as before they were run.  Consider using a [TemporaryFolder](http://blog.codeleak.pl/2015/01/testing-with-files-and-directories-in.html) rule to keep the state of the machine the same.